src/encoded/types/meta_workflow.py

```diff
@@ -1,11 +1,12 @@
 from snovault import collection, load_schema
 from encoded_core.types.meta_workflow import MetaWorkflow as CoreMetaWorkflow
-
+from .base import SUBMISSION_CENTER_MEMBER_CREATE_ACL
 from .base import Item as SMAHTItem
 
 
 @collection(
     name='meta-workflows',
+    acl=SUBMISSION_CENTER_MEMBER_CREATE_ACL,
```

Comment:
Does that mean that Submission Centers can create their own MetaWorkflows? Do we want that?

Reply:
I had thought about this... Since this doesn't involve actual instantiation of workflows it might be useful to allow centers to do this if they want to contribute back pipelines. Could always be added later though.

Reply:
Right, they can't create MetaWorkflowRuns. I think that's fine then.